Automattic/jetpack
# Sync: remove failed login attempts from sync

Related to p7rcWF-DL-p2

#### Changes proposed in this Pull Request:

* Remove failed login attempts from sync. 

#### Testing instructions:

* Fail login in notice that we don't that we are not sending the failied loggin attemt on the .com side. 


#### Proposed changelog entry for your changes:
